guides/source/caching_with_rails.md

```diff
@@ -5,34 +5,42 @@ Caching with Rails: An Overview
 
 This guide is an introduction to speeding up your Rails application with caching.
 
-Caching means to store content generated during the request-response cycle and
-to reuse it when responding to similar requests.
+After reading this guide, you will know about:
 
-Caching is often the most effective way to boost an application's performance.
-Through caching, websites running on a single server with a single database
-can sustain a load of thousands of concurrent users.
-
-Rails provides a set of caching features out of the box. This guide will teach
-you the scope and purpose of each one of them. Master these techniques and your
-Rails applications can serve millions of views without exorbitant response times
-or server bills.
-
-After reading this guide, you will know:
-
-* Fragment and Russian doll caching.
+* Solid Cache - a database-backed Active Support cache store.
+* Types of Caching Strategies including Fragment and Russian doll caching.
 * How to manage the caching dependencies.
 * Alternative cache stores.
+* Cache Keys
 * Conditional GET support.
 
 --------------------------------------------------------------------------------
 
-Basic Caching
--------------
+What is Caching?
+----------------
+
+Caching means storing content generated during the request-response cycle and
+reusing it when responding to similar requests. It's like keeping your favorite
+coffee mug right on your desk instead of in the kitchen cabinet — it’s ready when
+you need it, saving you time and effort.
+
+Caching is one of the most effective ways to boost an application's performance.
+It allows websites running on modest infrastructure — a single server with a
+single database — to sustain thousands of concurrent users.
+
+Rails provides a set of caching features out of the box which allows you to not
+only cache data, but also to tackle challenges like cache expiration, cache
+dependencies, and cache invalidation.
+
+In this guide, you can explore Rails' comprehensive caching strategies, from page
+caching to fragment caching and beyond. By using these techniques, you can
+empower your Rails applications to serve millions of views while keeping
+response times low and server bills manageable.
```

Comment:
Correct -  https://github.com/Ridhwana/rails/pull/11/files#r1879041717

"Since these features, Page Caching and Action Caching, have been deprecated since Rails 4, I thought we could remove this section and any references to it. Additionally, the [[actionpack-page_caching gem](https://github.com/rails/actionpack-page_caching)](https://github.com/rails/actionpack-page_caching) has not received updates in three years.

I can revert these changes if we don't want to remove it."